api-gateway/src/main/java/com/petclinic/bffapigateway/domainclientlayer/AuthServiceClient.java

```diff
@@ -59,13 +59,13 @@ public AuthServiceClient(
 //                .bodyToMono(UserDetails.class);
 //    }
 //
-//    public Flux<UserDetails> getUsers(String auth) {
-//        return webClientBuilder.build().get()
-//                .uri(authServiceUrl + "/users/withoutPages")
-//                .header("Authorization", auth)
-//                .retrieve()
-//                .bodyToFlux(UserDetails.class);
-//    }
+    public Flux<UserDetails> getUsers(String jwtToken) {
+        return webClientBuilder.build().get()
+                .uri(authServiceUrl + "/users/withoutPages")
+                .cookie("Bearer", jwtToken)
```

Comment:
```suggestion
                .cookie("Bearer", JwtToken)
```

I think it might be better to have a capital letter